examples/basic.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+
+import json
+import requests
+import sys
+
+sdk_key = sys.argv[1]
```

Comment:
super nit, we may check for number of arguments: len(sys.argv) and exit gracefully

Reply:
Great idea. I added the check and responded with the expected parameters. Ideally I'd use something like `argparse`, but for these examples I wanted to keep it as terse as possible.